storybook/stories/components/BlockImageCarousel/BlockImageCarousel.stories.js

```diff
@@ -32,6 +32,11 @@ export default {
           class: ['ThemeDark', 'sbdocs-preview-full-width'],
           color: '#000000',
         },
+        {
+          name: 'ThemeInternal',
+          class: ['ThemeInternal', 'sbdocs-preview-full-width'],
+          color: '#53C8ED',
+        },
```

Comment:
What does this addition do? This story is iframed, so the theme switcher doesn't affect it.

Reply:
ThemeInternal can still be tested via canvas view, so this allows for that. `sbdocs-preview-full-width` does get applied to the iframe (ThemeInternal does too, it just doesn't get applied to the contents inside it), so to prevent the iframe from changing width while switching themes, I included `sbdocs-preview-full-width`) as well.

Reply:
Sorry, though I tried the Canvas view, but apparently I did not!